### PR TITLE
Golden metrics configuration added for GCPSpannerDatabase

### PR DESCRIPTION
**Pull request by fprats**
### Relevant information

Describe what you have done and any details that you think are relevant or that you might want to discuss with us. 
Added configuration for default golden metrics through the golden-metrics.yml file

### Checklist

* [ ] I've read the guidelines and understand the acceptance criteria.
* [ ] The value of the attribute marked as `identifier` will be unique and valid. 
* [ ] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

### CODE REVIEW

File: definitions/infra-gcpspannerdatabase/golden_metrics.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Requests:
```
Comment (naxhh) [inline definitions/infra-gcpspannerdatabase/golden_metrics.yml:1]: nit

```suggestion
requests:
```

File: definitions/infra-gcpspannerdatabase/golden_metrics.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  title: Number of requests
+  query:
+    select: sum(`api.Requests`)
+    from: GcpSpannerDatabaseSample
+    facet: entityName
+    eventId: entityGuid
+ErrorRate:
```
Comment (naxhh) [inline definitions/infra-gcpspannerdatabase/golden_metrics.yml:8]: nit:
```suggestion
ErrorRate:
```

**Comment by fprats** [comment]
@naxhh Suggested changes committed, many thanks. Could you take a look again?

**Review by naxhh — APPROVED** [review]

**Comment by naxhh** [comment]
Thanks for the contribution!

**Pull request merged**